published/external/xdpapi.h

```diff
@@ -28,9 +33,12 @@ typedef enum _XDP_CREATE_PROGRAM_FLAGS {
 DEFINE_ENUM_FLAG_OPERATORS(XDP_CREATE_PROGRAM_FLAGS);
 C_ASSERT(sizeof(XDP_CREATE_PROGRAM_FLAGS) == sizeof(UINT32));
 
+#if defined(_KERNEL_MODE)
+
 typedef
-HRESULT
+XDP_STATUS
 XDP_CREATE_PROGRAM_FN(
```

Comment:
every function defined in this header and afxdp.h needs an IRQL annotation

Reply:
I agree

Reply:
I saw other IRQL annotations, but it is really hard to know which level is appropriate.

Reply:
asking AI about their thought process.